src/apps/slackapp/slackapp/bolt_app.py

```diff
@@ -46,22 +48,32 @@ def hello_command(ack, body):
     ack(f"Hi, <@{user_id}>!")
 
 
-def contains_verbose(query: str) -> bool:
-    """looks for -verbose in the question and returns True or False"""
-    return "-verbose" in query.lower()
+def process_chat_history(messages: List[dict]) -> List[BaseMessage]:
```

Comment:
Can we use a more descriptive function name here? E.g. `cleaned_and_classified_messages`